lms/djangoapps/commerce/models.py

```diff
@@ -1,3 +1,19 @@
-"""
-This file is intentionally empty. Django 1.6 and below require a models.py file for all apps.
-"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class CommerceConfiguration(ConfigurationModel):
+    """ Commerce configuration """
+
```

Comment:
I know I've asked about this before, but I just want to ask again if this is the right location/time to add a foreign key back to the Site.